.github/workflows/build.yml

```diff
@@ -3,54 +3,64 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-name: Go
+name: build
 on: [push, pull_request]
 jobs:
-
   unitTest:
-    name: Unit Test
+    name: Unit test
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest]
+        os: [ubuntu-18.04, macOS-10.14]
     steps:
 
-    - name: Set up Go 1.13
+    - name: Setup Go 1.13
       uses: actions/setup-go@v1
       with:
         go-version: 1.13
       id: go
 
     - uses: actions/checkout@v1
 
-    - name: Unit Test
-      timeout-minutes: 5
+    - name: Run unit test
+      timeout-minutes: 10
       run: make unit-test
 
+    # TODO: need secret for Codecov from Actions
```

Comment:
Doesn't look like we have the necessary option under this project's Settings tab: https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables

Reply:
Yup - that's why I left CircleCI unit test step enabled.